docs/c-runtime-library/locale-names-languages-and-country-region-strings.md

```diff
@@ -37,6 +37,22 @@ _wsetlocale(LC_ALL, L"de-DE");
 _wsetlocale(LC_ALL, L"LC_MONETARY=en-GB;LC_TIME=es-ES");
 ```
 
+
+## UTF-8 Support
+
+Starting in Windows 10 build 17134 (April 2018 Update), the Universal C Runtime supports using a UTF-8 code page. This means that narrow (char) strings passed to C runtime functions will expect strings in the UTF-8 encoding and properly understand them. To enable this, use "UTF-8" as the code page when using setlocale. For example, [setlocale(LC_ALL, ".utf8")](reference/setlocale-wsetlocale.md) will use the current ACP for the locale and UTF-8 for the code page.
+
+After calling setlocale(LC_ALL, ".UTF8"), you may pass "😊" to mbtowcs and it will be properly translated to a wide string, whereas previously there was not a locale setting available to do this.
+
+UTF-8 mode is also enabled for functions that have historically translated narrow strings using the default Windows ANSI code page (ACP). For example, calling [_mkdir("😊")](reference/mkdir-wmkdir.md) while using a UTF-8 code page will correctly produce a directory with that emoji as the folder name, instead of requiring the ACP to be changed to UTF-8 prior to running your program. Likewise, calling [_getcwd()](reference/getcwd-wgetcwd.md) inside of that folder will return a UTF-8 encoded string. For behavior compatibility, the ACP is still used if the C locale code page is not set to UTF-8.
+
+There are some aspects of the C Runtime that are not able to use UTF-8 because they are set during program startup and must use the default Windows ANSI code page (ACP): [__argv](argc-argv-wargv.md), [_acmdln](acmdln-tcmdln-wcmdln.md), and [_pgmptr](pgmptr-wpgmptr.md).
+
+Previous to this support, [mbrtoc16, mbrtoc32](reference/mbrtoc16-mbrtoc323.md), [c16rtomb, and c32rtomb](reference/c16rtomb-c32rtomb1.md) existed to translate between UTF-8 narrow strings, UTF-16 (same encoding as wchar_t on Windows platforms) and UTF-32. For behavior compatibility reasons, these APIs still only translate to and from UTF-8 and not the code page set via setlocale.
```

Comment:
can whack 'behavior'. Just 'compatibility reasons' will do.  You should also bold style setlocale

Reply:
To be compatible with c-runtime doc styling, and to keep machine translation from changing things it shouldn't, let's use this style for anything code like wchar_t, functions, etc. **`code thingy`**  (basically, bold with code style inside)